python/CommonFramework.cpp

```diff
@@ -32,7 +32,12 @@ PythonBase::PythonBase() :
     m_program_name(""),
 #endif
     m_python_module_error(nullptr)
-{}
+{
+#if defined(FREEORION_WIN32)
```

Comment:
would prefer to initialize at definition in the header, and to avoid the extra `#if`